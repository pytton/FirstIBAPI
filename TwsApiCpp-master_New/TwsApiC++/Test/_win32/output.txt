
Time: ,Contract: ,Bid: ,Ask: ,Bid size: ,Ask size: 
327,EUR,0,0,0,0
330,GBP,0,0,0,0
333,SPY,0,0,0,0
336,DIA,0,0,0,0
340,IWM,0,0,0,0
649,EUR,0,0,0,0
663,GBP,0,0,0,0
674,SPY,0,0,0,0
683,DIA,0,0,0,0
693,IWM,0,0,0,0
1008,EUR,1.09295,1.093,8866000,4683000
1026,GBP,0,0,0,0
1037,SPY,0,0,0,0
1050,DIA,-1,-1,0,0
1062,IWM,-1,-1,0,0
1350,EUR,1.09305,1.0931,9066000,2866000
1362,GBP,1.5267,1.5268,3820000,820000
1371,SPY,0,0,0,0
1377,DIA,-1,-1,0,0
1382,IWM,-1,-1,0,0
1641,EUR,1.09305,1.0931,9066000,2366000
1644,GBP,1.5267,1.5268,3820000,820000
1646,SPY,0,0,0,0
1648,DIA,-1,-1,0,0
1649,IWM,-1,-1,0,0
1903,EUR,1.09305,1.0931,10066000,1683000
1906,GBP,1.5267,1.5268,3820000,820000
1908,SPY,0,0,0,0
1910,DIA,-1,-1,0,0
1911,IWM,-1,-1,0,0
2166,EUR,1.09305,1.0931,10066000,1000000
2169,GBP,1.5267,1.5268,3820000,820000
2172,SPY,0,0,0,0
2173,DIA,-1,-1,0,0
2175,IWM,-1,-1,0,0
2429,EUR,1.09305,1.0931,10066000,1000000
2432,GBP,1.5267,1.5268,3820000,820000
2435,SPY,0,0,0,0
2436,DIA,-1,-1,0,0
2438,IWM,-1,-1,0,0
2693,EUR,1.09305,1.0931,10066000,1000000
2695,GBP,1.5267,1.5268,3820000,820000
2698,SPY,0,0,0,0
2699,DIA,-1,-1,0,0
2701,IWM,-1,-1,0,0
2956,EUR,1.09305,1.0931,10066000,1000000
2958,GBP,1.5267,1.5268,3820000,820000
2961,SPY,0,0,0,0
2962,DIA,-1,-1,0,0
2964,IWM,-1,-1,0,0
3227,EUR,1.09305,1.0931,10066000,1000000
3239,GBP,1.5267,1.5268,3820000,1320000
3249,SPY,0,0,0,0
3254,DIA,-1,-1,0,0
3259,IWM,-1,-1,0,0
3528,EUR,1.09305,1.0931,10066000,1000000
3539,GBP,1.5267,1.5268,3820000,1320000
3552,SPY,0,0,0,0
3560,DIA,-1,-1,0,0
3568,IWM,-1,-1,0,0
3829,EUR,1.09305,1.0931,10066000,1000000
3831,GBP,1.5267,1.5268,3820000,1451000
3834,SPY,0,0,0,0
3835,DIA,-1,-1,0,0
3837,IWM,-1,-1,0,0
4091,EUR,1.09305,1.0931,10066000,1000000
4094,GBP,1.5267,1.5268,3820000,1451000
4096,SPY,0,0,0,0
4097,DIA,-1,-1,0,0
4099,IWM,-1,-1,0,0
4353,EUR,1.09305,1.0931,10066000,1000000
4356,GBP,1.5267,1.5268,3820000,1451000
4359,SPY,0,0,0,0
4360,DIA,-1,-1,0,0
4361,IWM,-1,-1,0,0
4626,EUR,1.09305,1.0931,10066000,1000000
4637,GBP,1.5267,1.5268,3820000,1451000
4645,SPY,0,0,0,0
4649,DIA,-1,-1,0,0
4653,IWM,-1,-1,0,0
4918,EUR,1.09305,1.0931,10066000,1000000
4927,GBP,1.5267,1.5268,3820000,1451000
4934,SPY,0,0,0,0
4940,DIA,-1,-1,0,0
4946,IWM,-1,-1,0,0
5204,EUR,1.09305,1.0931,10066000,1000000
5207,GBP,1.5267,1.5268,3820000,951000
5209,SPY,0,0,0,0
5210,DIA,-1,-1,0,0
5212,IWM,-1,-1,0,0
5466,EUR,1.09305,1.0931,10066000,1000000
5469,GBP,1.5267,1.5268,3820000,820000
5471,SPY,0,0,0,0
5473,DIA,-1,-1,0,0
5474,IWM,-1,-1,0,0
5738,EUR,1.09305,1.0931,10066000,1000000
5749,GBP,1.5267,1.5268,3820000,820000
5756,SPY,0,0,0,0
5762,DIA,-1,-1,0,0
5765,IWM,-1,-1,0,0
6029,EUR,1.09305,1.0931,10066000,1000000
6042,GBP,1.5267,1.5268,3820000,820000
6049,SPY,-1,-1,0,0
6054,DIA,-1,-1,0,0
6061,IWM,-1,-1,0,0
6320,EUR,1.09305,1.0931,10066000,1000000
6323,GBP,1.5267,1.5268,3820000,820000
6325,SPY,-1,-1,0,0
6326,DIA,-1,-1,0,0
6328,IWM,-1,-1,0,0
6582,EUR,1.09305,1.0931,10066000,1000000
6584,GBP,1.5267,1.5268,3820000,820000
6587,SPY,-1,-1,0,0
6589,DIA,-1,-1,0,0
6591,IWM,-1,-1,0,0
6845,EUR,1.09305,1.0931,10066000,1000000
6848,GBP,1.5267,1.5268,3820000,820000
6857,SPY,-1,-1,0,0
6859,DIA,-1,-1,0,0
6861,IWM,-1,-1,0,0
7117,EUR,1.09305,1.0931,10066000,1000000
7120,GBP,1.5267,1.5268,3820000,820000
7123,SPY,-1,-1,0,0
7124,DIA,-1,-1,0,0
7130,IWM,-1,-1,0,0
7396,EUR,1.09305,1.0931,10066000,1000000
7408,GBP,1.5267,1.5268,3820000,820000
7414,SPY,-1,-1,0,0
7417,DIA,-1,-1,0,0
7420,IWM,-1,-1,0,0
7689,EUR,1.09305,1.0931,10066000,1000000
7702,GBP,1.5267,1.5268,3820000,820000
7712,SPY,-1,-1,0,0
7717,DIA,-1,-1,0,0
7724,IWM,-1,-1,0,0
7995,EUR,1.09305,1.0931,10066000,1000000
8007,GBP,1.5267,1.5268,3820000,820000
8016,SPY,-1,-1,0,0
8022,DIA,-1,-1,0,0
8028,IWM,-1,-1,0,0
8301,EUR,1.09305,1.0931,16733000,1000000
8310,GBP,1.5267,1.52685,5140000,12151000
8317,SPY,-1,-1,0,0
8324,DIA,-1,-1,0,0
8329,IWM,-1,-1,0,0
8599,EUR,1.0931,1.09315,1000000,13066000
8610,GBP,1.5267,1.52685,5140000,10951000
8619,SPY,-1,-1,0,0
8626,DIA,-1,-1,0,0
8632,IWM,-1,-1,0,0
8897,EUR,1.0931,1.09315,1000000,12066000
8910,GBP,1.5267,1.52685,5140000,9951000
8919,SPY,-1,-1,0,0
8925,DIA,-1,-1,0,0
8931,IWM,-1,-1,0,0
9204,EUR,1.09305,1.0931,17733000,1000000
9216,GBP,1.5267,1.52685,5140000,9951000
9227,SPY,-1,-1,0,0
9233,DIA,-1,-1,0,0
9239,IWM,-1,-1,0,0
9507,EUR,1.09305,1.0931,17733000,1000000
9518,GBP,1.5267,1.52685,5140000,9951000
9528,SPY,-1,-1,0,0
9534,DIA,-1,-1,0,0
9544,IWM,-1,-1,0,0
9813,EUR,1.09305,1.0931,17733000,1000000
9825,GBP,1.5267,1.52685,5140000,7951000
9835,SPY,-1,-1,0,0
9841,DIA,-1,-1,0,0
9846,IWM,-1,-1,0,0
10115,EUR,1.09305,1.0931,17733000,1000000
10127,GBP,1.5267,1.52685,5140000,7951000
10134,SPY,-1,-1,0,0
10141,DIA,-1,-1,0,0
10148,IWM,-1,-1,0,0
10418,EUR,1.09305,1.0931,17733000,1000000
10430,GBP,1.5267,1.52685,5140000,7951000
10438,SPY,-1,-1,0,0
10443,DIA,-1,-1,0,0
10448,IWM,-1,-1,0,0
10717,EUR,1.09305,1.0931,17733000,1000000
10727,GBP,1.5267,1.52685,5140000,7951000
10735,SPY,-1,-1,0,0
10741,DIA,-1,-1,0,0
10747,IWM,-1,-1,0,0
11014,EUR,1.09305,1.0931,17733000,1000000
11025,GBP,1.5267,1.52685,5140000,7951000
11033,SPY,-1,-1,0,0
11039,DIA,-1,-1,0,0
11044,IWM,-1,-1,0,0
11313,EUR,1.09305,1.0931,18733000,2000000
11325,GBP,1.5267,1.52685,5140000,7951000
11333,SPY,-1,-1,0,0
11340,DIA,-1,-1,0,0
11346,IWM,-1,-1,0,0
11613,EUR,1.09305,1.0931,18066000,2366000
11626,GBP,1.5267,1.52685,6140000,7951000
11634,SPY,-1,-1,0,0
11641,DIA,-1,-1,0,0
11647,IWM,-1,-1,0,0
11918,EUR,1.09305,1.0931,17866000,2366000
11929,GBP,1.5267,1.5268,5140000,820000
11936,SPY,-1,-1,0,0
11942,DIA,-1,-1,0,0
11947,IWM,-1,-1,0,0
12214,EUR,1.09305,1.0931,17866000,2366000
12227,GBP,1.5267,1.5268,5140000,1320000
12235,SPY,-1,-1,0,0
12242,DIA,-1,-1,0,0
12247,IWM,-1,-1,0,0
12516,EUR,1.09305,1.0931,17866000,2366000
12527,GBP,1.5267,1.52685,5140000,13020000
12535,SPY,-1,-1,0,0
12542,DIA,-1,-1,0,0
12545,IWM,-1,-1,0,0
12812,EUR,1.09305,1.0931,18066000,2366000
12825,GBP,1.5267,1.52685,5140000,13151000
12834,SPY,-1,-1,0,0
12841,DIA,-1,-1,0,0
12847,IWM,-1,-1,0,0
13117,EUR,1.09305,1.0931,18066000,2366000
13127,GBP,1.5267,1.52685,5140000,11151000
13134,SPY,-1,-1,0,0
13141,DIA,-1,-1,0,0
13145,IWM,-1,-1,0,0
13414,EUR,1.09305,1.0931,17066000,2366000
13424,GBP,1.5267,1.52685,5140000,11151000
13433,SPY,-1,-1,0,0
13440,DIA,-1,-1,0,0
13446,IWM,-1,-1,0,0
13713,EUR,1.09305,1.0931,16066000,2366000
13726,GBP,1.5267,1.52685,5140000,11151000
13734,SPY,-1,-1,0,0
13740,DIA,-1,-1,0,0
13746,IWM,-1,-1,0,0
14017,EUR,1.09305,1.0931,16066000,2366000
14027,GBP,1.5267,1.52685,5140000,11151000
14035,SPY,-1,-1,0,0
14041,DIA,-1,-1,0,0
14046,IWM,-1,-1,0,0
14314,EUR,1.09305,1.0931,16066000,2366000
14327,GBP,1.5267,1.52685,5140000,11151000
14335,SPY,-1,-1,0,0
14342,DIA,-1,-1,0,0
14347,IWM,-1,-1,0,0
14613,EUR,1.09305,1.0931,16066000,2366000
14622,GBP,1.5267,1.52685,5140000,11151000
14631,SPY,-1,-1,0,0
14639,DIA,-1,-1,0,0
14644,IWM,-1,-1,0,0
