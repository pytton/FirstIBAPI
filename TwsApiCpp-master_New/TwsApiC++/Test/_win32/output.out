time: 259

SPY
ask: 1.0917
bid: 1.09165
ask size: 1183000
bid size: 11683000
time: 265

DIA
ask: 1.5264
time: 517

SPY
ask: 1.09175
bid: 1.0917
ask size: 1700000
bid size: 1000000
time: 520

DIA
ask: 1.5264
time: 773

SPY
ask: 1.09175
bid: 1.0917
ask size: 10883000
bid size: 2500000
time: 776

DIA
ask: 1.52635
time: 1029

SPY
ask: 1.09175
bid: 1.5262
ask size: 2200000
bid size: 10683000
time: 1033

DIA
ask: 1.5262
time: 1286

SPY
ask: 1.09175
bid: 1.5262
ask size: 8700000
bid size: 6820000
time: 1291

DIA
ask: 1.5262
time: 1544

SPY
ask: 1.0918
bid: 1.5262
ask size: 13983000
bid size: 12883000
time: 1548

DIA
ask: 1.5263
time: 1802

SPY
ask: 1.0918
bid: 1.09175
ask size: 2783000
bid size: 5183000
time: 1806

DIA
ask: 1.5263
time: 2059

SPY
ask: 1.0918
bid: 1.09175
ask size: 2283000
bid size: 7683000
time: 2063

DIA
ask: 1.5263
time: 2321

SPY
ask: 1.0918
bid: 1.09175
ask size: 2183000
bid size: 7683000
time: 2340

DIA
ask: 1.5263
time: 2599

SPY
ask: 1.0918
bid: 1.09175
ask size: 2183000
bid size: 7683000
time: 2614

DIA
ask: 1.5263
